spec/models/user_spec.rb

```diff
@@ -27,6 +27,7 @@
 #  organization_admin     :boolean
 #  name                   :string           default("CHANGEME"), not null
 #  super_admin            :boolean          default(FALSE)
+#  last_request_at        :datetime
```

Comment:
Why was this needed?

Reply:
This was added automatically when I ran annotate

Reply:
Should I revert this ?